### ISSUE TITLE
stay on top

### ISSUE THREAD

**Issue by Tecknosa** [jaden/SnapTimer#31]
the stay on top feature isnt working and not sure how to fix

**Comment by xmatic** [comment]
> the stay on top feature isnt working and not sure how to fix

Agreed. It is iffy. Sometimes it works, sometimes it doesn't work. 

Would like a permanent fix because it makes this otherwise awesome portable app feel irritating.
